### PR TITLE
Explicitly pass { passive: false } for wheel/mousewheel events (T724265)

### PR DESCRIPTION
**Pull request by IgnatovDan**

### CODE REVIEW

File: js/events/core/events_engine.js [hunk 1]
```diff
@@ -183,7 +180,7 @@ var getHandlersController = function(element, eventName) {
             if(shouldAddNativeListener) {
                 eventData.nativeHandler = getNativeHandler(eventName);
 
-                if(eventName === passiveListenerEvents.nativeEventName) {
+                if(forcePassiveFalseEventNames.includes(eventName)) {
```
Comment (alexander-kotov-dx) [inline js/events/core/events_engine.js]: Does array.includes work for IE11?

**Comment by alexander-kotov-dx** [comment]
Can we add some test cases?

File: js/events/core/events_engine.js (hunk 1 above)
Comment (IgnatovDan) [reply js/events/core/events_engine.js]: No. I used indexOf instead.

File: js/integration/jquery/events.js [hunk 2]
```diff
@@ -9,12 +9,13 @@ if(useJQuery) {
...
-            domAdapter.listen(this, passiveListenerEventName, handler, { passive: false });
-        }
-    };
+    eventsEngine.forcePassiveFalseEventNames.forEach(function(eventName) {
+        jQuery.event.special[eventName] = {
+            setup: function(data, namespaces, handler) {
+                domAdapter.listen(this, eventName, handler, { passive: false });
+            }
+        };
+    });
```
Comment (jsdmitry) [inline js/integration/jquery/events.js:18]: Where tests on this functionality?

Comment (IgnatovDan) [reply js/integration/jquery/events.js:18]: There was one simple test for this method, see [Fix Illegal Invocation errors...](https://github.com/IgnatovDan/DevExtreme/commit/42077826aafa30b216b6203acdbc387de19cfe04)
I added a new one for the complete event usage scenario and for several events: [Test how our code calls the add/removeEventListener methods with jQue…](https://github.com/DevExpress/DevExtreme/pull/7304/commits/afde85d4cc115036565f8c2017027409a60e92de)

**Review by jsdmitry — APPROVED** [review]

**Review by alexander-kotov-dx — APPROVED** [review]

**Pull request merged**